ui-admin/src/study/surveys/SurveyTemplates.tsx

```diff
@@ -0,0 +1,34 @@
+import { Survey } from '@juniper/ui-core'
+
+
+const DEFAULT_TEMPLATE = '{"pages":[{"elements":[]}]}'
+const OUTREACH_SCREENER_TEMPLATE = (survey: Survey) => `
+{"pages":[{"elements":[{"type":"html","name":"outreach_content_${survey.stableId}"}]}]}
+`
+// eslint-disable-next-line max-len
+const DOC_REQUEST_TEMPLATE = (survey: Survey) => `
+{
+"pages":
+  [
+    {"elements":
+      [
+        {
+          "type":"html",
+          "name":"doc_request_content_${survey.stableId}",
+          "html": "<h3>${survey.name}</h3><p>Please upload your document here.</p>"
+        }
+      ]
+    }
+  ]
+}
+`
+
+export const getSurveyContentTemplate = (survey: Survey, isScreener: boolean): string => {
+  if (survey.surveyType === 'OUTREACH' && !isScreener) {
+    return OUTREACH_SCREENER_TEMPLATE(survey)
+  }
+  if (survey.surveyType === 'DOCUMENT_REQUEST') {
+    return DOC_REQUEST_TEMPLATE(survey)
+  }
+  return DEFAULT_TEMPLATE
+}
```

Comment:
I wanted to formalize the concept of a `survey template` without pushing it to the backend. I'm not super happy with `isScreener` but it's a start.